client/src/components/Widget/Widget.tsx

```diff
@@ -23,40 +23,40 @@ const Widget: FunctionComponent<WidgetProps> = ({ color, data }) => {
     <S.Widget color={color}>
       <S.WidgetHeader color={color}>
         <Flex justifySpaceBetween>
-          <time dateTime={timeStamp.day}>{timeStamp.day}</time>
-          <time dateTime={timeStamp.time}>{timeStamp.time}</time>
+          <time dateTime={timeStamp?.day}>{timeStamp?.day}</time>
+          <time dateTime={timeStamp?.time}>{timeStamp?.time}</time>
         </Flex>
       </S.WidgetHeader>
       <S.WidgetBody>
         <Flex>
           <S.WidgetDetails>
-            <S.WidgetTemp>{data.temperature}°</S.WidgetTemp>
+            <S.WidgetTemp>{data?.temperature}°</S.WidgetTemp>
             <S.WidgetDetailsItem color={color}>
-              <span>Real Feel</span> {data.feelsLikeTemp}°
+              <span>Real Feel</span> {data?.feelsLikeTemp}°
             </S.WidgetDetailsItem>
             <S.WidgetDetailsItem color={color}>
-              <span>Wind</span> {data.windDirString}, speed {data.windSpeed}&nbsp;m/s, gust{' '}
-              {data.windGust} m/s
+              <span>Wind</span> {data?.windDirString}, speed {data?.windSpeed}&nbsp;m/s, gust{' '}
```

Comment:
Do you need `{' '}`

Reply:
Apparently, yes. I need a space between `gust` and `{data.windGust} m/s`, but the latter goes to next line after file save, so Prettier adds this `{' '}`.